### PR TITLE
Add className attribute to TextField of DatePicker

### PR DESCRIPTION
**Pull request by phtucker**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

TextField element of DatePicker was not picking up styles. This was due to the className attribute not being present on the TextField. This PR adds the className attribute and appropriate value and has been verified as solving the aforementioned issue.

#### Focus areas to test

DatePicker

### CODE REVIEW

File: common/changes/office-ui-fabric-react/datePicker_textField_className_2019-01-17-19-21.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "added className attribute to TextField of DatePicker",
```
Comment (Vitalius1) [inline common/changes/office-ui-fabric-react/datePicker_textField_className_2019-01-17-19-21.json]: This comment is used for official release notes. I would suggest something like this:
```suggestion
      "comment": "DatePicker: adds missing className attribute to the `TextField` of the control.",
```

**Comment by Vitalius1** [comment]
@lorejoh12 can you please take a look? Thanks!

**Comment by Vitalius1** [comment]
@phtucker You would have to run `npm run update-snapshots` from the root of the repo to update a failing snapshot and then commit the change and push to your PR.

**Comment by phtucker** [comment]
I had updated the snapshots with the initial commit. Will do so again tomorrow. Thanks.

**Comment by phtucker** [comment]
Updated snapshots, checks now passing.

**Comment by Vitalius1** [comment]
@lorejoh12 can someone on the Calendar team approve this? Thanks.

**Pull request merged**